src/pages/AllUsersPage/NotFound/NotFound.styles.scss

```diff
@@ -0,0 +1,297 @@
+@import url(https://fonts.googleapis.com/css?family=Ubuntu);
+//variables
+$purple: #28254C;
+$l-purple: #332F63;
+$t-purple: #8C8AA7;
```

Comment:
Why are we adding the code for the same component again?

Reply:
It just got deleted from my side so if I thought if you merge it, this file will get deleted from your github too so that's why created it and copy pasted it.

Reply:
@BurhanRaja But I can already see the other component as well, it inside ./pages

Reply:
I realised it now. Sorry.